src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIOffenderService.kt

```diff
@@ -128,36 +126,30 @@ class CommunityAPIOffenderService(
       .bodyToMono(OffenderIdentifiersResponse::class.java)
   }
 
-  fun getResponsibleOfficer(crn: String): ResponsibleOfficer {
-    val offenderManagersPath = UriComponentsBuilder.fromPath(offenderManagersLocation)
+  fun getResponsibleOfficerDetails(crn: String): OfficerDetails? {
+    val officerDetailsPath = UriComponentsBuilder.fromPath(responsibleOfficerLocation)
       .buildAndExpand(crn)
       .toString()
 
-    val responsibleOfficers = communityApiClient.get(offenderManagersPath)
+    val responsibleOfficer = communityApiClient.get(officerDetailsPath)
```

Comment:
Should this be officerDetails instead of responsibleOfficer since we don't know whether they are the responsible officer yet at this point?

Reply:
Agree, will update it. thanks